main.js

```diff
@@ -0,0 +1,23 @@
+const handleBarsAppend = (data) => {
+  const source = $('#people-template').html();
+  const template = Handlebars.compile(source);
+  const newHTML = template(data);
+
+  $('body').append(newHTML);
```

Comment:
it's better to create a container div for any appending than to append something to the <body>